src/components/CommentInfo/CommentInfo.scss

```
@@ -0,0 +1,50 @@
+@import "../../utils/variables";
+
+.comments {
+  border-radius: 15px;
+  margin-top: 30px;
+  background-color: $blockCommentBg;
+  padding: 15px;
```

Remember about proper style order, fix it everywhere
```suggestion
  margin-top: 30px;
  padding: 15px;
  border-radius: 15px;
  background-color: $blockCommentBg;
```